src/WPML_OptionsManager.php

```diff
@@ -536,10 +538,78 @@ public function LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
+            <?php $this->displayMP3Banner() ?>
         </div>
         <?php
     }
 
+    private function displayMP3Banner() {
+        ?>
+        <div style="background: #fff;border-left: 4px solid #fff;border-radius: 10px;box-shadow: 0 4px 35px rgba(195, 65, 2, .2);clear: both;margin-bottom: 15px;padding: 20px;position: relative;">
+            <h3><?php _e( 'Reliable and beautiful emails by MailPoet!', 'wp-mail-logging' );?></h3>
+            <?php
+              if (rand(1, 2) === 1) {
+                    $this->displayMP3BannerVersionA();
+              } else {
+                    $this->displayMP3BannerVersionB();
+              }
+            ?>
+        </div>
+        <?php
+    }
+
+    private function displayMP3BannerVersionA() {
+        ?>
+            <ul style="list-style-type:disc">
+                <li><?php _e( '50 email templates to chose from', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Fun email designer', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Automated email marketing', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'WooCommerce emails', 'wp-mail-logging' );?></li>
+                <li><?php _e( '99.1% success email deliverability', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Fast and friendly support', 'wp-mail-logging' );?></li>
+                <li><?php _e( 'Over 100,000 active users', 'wp-mail-logging' );?></li>
+            </ul>
+            <a
+                class="button button-primary"
+                href="?page=wpml_plugin_log&redirect=free-plan&ref=wml_1"
+                target="_blank"
+            >
+                <?php _e( 'Try MailPoet for free', 'wp-mail-logging' );?>
+            </a>
+            <p>
+                <?php _e( 'Testimonial', 'wp-mail-logging' );?>:
+                <i>
+                    <?php _e( 'Thanks for this awesome plugin, love love love how it integrates with WordPress. I seriously spent days if not weeks on Mailchimp, and still haven’t been able to do what I did on MailPoet in 1 hour!', 'wp-mail-logging' );?>
+                </i>
+                — Kida Shey
+            </p>
+        <?php
+    }
+
+    private function displayMP3BannerVersionB() {
+        ?>
+        <p>
+            <?php _e( 'Create beautiful email campaigns and reach your audience in a breeze with the MailPoet  plugin. MailPoet is used by over 300,000 website owners making it the most popular email marketing solution in WordPress. Send beautiful newsletter, notify your readers about last articles and increase your WooCommerce sales!', 'wp-mail-logging' );?>
+            <a href="?page=wpml_plugin_log&redirect=free-plan&ref=wml_2" target="_blank">
+                <?php _e( 'Discover MailPoet for free', 'wp-mail-logging' );?>
```

Comment:
Should this be a blue button?

Reply:
The spec says link so I am not sure

Reply:
The main CTA should be a styled as a blue button.